Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Function.cs

```diff
@@ -20,19 +22,23 @@ public Functions()
     /// <summary>
     /// A Lambda function to respond to HTTP Get methods from API Gateway
     /// </summary>
-    /// <param name="request"></param>
-    /// <returns>The API Gateway response.</returns>
-    public APIGatewayProxyResponse Get(APIGatewayProxyRequest request, ILambdaContext context)
+    /// <remarks>
+    /// This uses the Lambda Annotations programming model to bridge the gap between 
```

Comment:
Can you add a link to the README also call out that it takes care of syncing with CloudFormation. It is important users realize this change in behavior.

Reply:
I added both:
* A link to the GitHub readme
* A line about template syncing.

I couldn't figure out how to do a `<see>` link to the local readme, I think because it's not a .NET object.